docker-compose.override.yml

```diff
@@ -20,6 +20,9 @@ services:
     volumes:
       - postgres-volume:/var/lib/postgresql/data
       - ./scripts/test.sql:/docker-entrypoint-initdb.d/test.sql
+      - ./build/logs/postgres:/var/lib/postgresql/data/logs
```

Comment:
I'm assuming if the `logs` folder does not exist on the developer's machine then it will get created?